rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -406,7 +406,7 @@ void log_frame::RepaintTextColors()
 {
 	// Backup old colors
 	QColor old_color_stack{ m_color_stack };
-	QList<QColor> old_color{ m_color };
+	//QList<QColor> old_color{ m_color };
```

Comment:
you can safely removed this whole backup old colors section, so that "// Get text color ... " is the first line in the function

Reply:
Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/470e6d9a76bedeb81a483de118da4c140f10babf